### DISCUSSION TITLE
When i use controller for clicking circles in osu standard, the left button doesnt work (im on lazer mobile)

### DISCUSSION THREAD

**Discussion by CoolGuy12490** [ppy/osu#19671] [Q&A] [answered]
I use a Samsung Galaxy Tab A7 running Android 12 (One UI 4.1) if it helps.

I wanted to drop this possible bug with the game or OS because I don't have access to log files. If this is really a bug, suggest some files apps so I can get access to the log files to report this where it probably should've gone.

**Comment by naipofo** [comment, marked as answer]
From what I understand, osu is treating you holding your finger on the screen as holding the left button. Try to toggle `Disable mouse buttons during gameplay`.

**Reply by peppy** [reply]
This is correct.
